packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -432,7 +412,7 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       }),
       updateFailedAttempts: assign({
         failedAttempts: (context) => {
-          return context.failedAttempts + 1;
+          return context.failedAttempts! + 1;
```

Comment:
Instead of using the non-null assertion operator here we could instead typecast `context` like so. What does everyone think?

```
context as any as HydratedLivenessContext
```